content/blog/one-year-of-excalidraw/index.md

```diff
@@ -0,0 +1,87 @@
+---
+title: One Year of Excalidraw
+date: 2020-12-25
+author: Excalidraw Team
+link: https://github.com/orgs/excalidraw/people
+image: og.png
+---
+
+> It's been kind of a different year, but it was the first year and pretty amazing for [Excalidraw](https://excalidraw.com).
+
+<!-- end -->
+
+Excalidraw started as a way to procrastinate on January 1st, 2020 and ended up being a fully fledged product only one year later! It unfortunately became a core part of many people's workflows due to the pandemic. In this post, we'll go over the most important features that made Excalidraw what it is.
```

Comment:
```suggestion
Excalidraw started as a way to procrastinate on January 1st, 2020 and ended up being a fully fledged product only one year later! It ultimately became a core part of many people's workflows due to the pandemic. In this post, we'll go over the most important features that made Excalidraw what it is.
```

Reply:
I went through several iterations, but nothing sounded right. I think we should just drop the sentence altogether as it doesn't really stand on its own, nor fit in the context of the paragraph, and puts the pandemic in the spotlight which is never good.

Damn, we're spending a lot of time on this one sentence :sweat_smile: 

(Come for the pandemic, stay for the product!)